corehq/apps/users/migrations/0001_add_location_permission.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-03-09 02:05
+from __future__ import unicode_literals
+from corehq.sql_db.operations import HqRunPython
+from corehq.apps.users.management.commands.add_location_permission import update_all_roles
+
+from django.db import migrations
+
+
+def migrate(apps, schema_editor):
+    update_all_roles()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        HqRunPython(migrate),
```

Comment:
Do you think it would be hard to add `edit_locations` to the Permissions schema and a management command in 1 PR, then run the management command in a shell and then have a second PR that adds the permission checking?